bin/flutter

```diff
@@ -146,6 +150,8 @@ SNAPSHOT_PATH="$FLUTTER_ROOT/bin/cache/flutter_tools.snapshot"
 STAMP_PATH="$FLUTTER_ROOT/bin/cache/flutter_tools.stamp"
 SCRIPT_PATH="$FLUTTER_TOOLS_DIR/bin/flutter_tools.dart"
 DART_SDK_PATH="$FLUTTER_ROOT/bin/cache/dart-sdk"
+DART_2_NATIVE="$FLUTTER_ROOT/bin/cache/dart-sdk/bin/dart2native"
```

Comment:
What's the difference between `dart2native` and `dart2aot`?

Reply:
dart2native is built by default in our build of the dart sdk, dart2aot doesn't seem to exist. It might be the case that one will replace the other. I'll have to follow up with @mit-mit to see what the right artifacts are

Reply:
Sorry, I missed this question. `dart2native` is the new tool that supports both self-contained executables and AOT snapshots. `dart2aot` will get removed in our next stable release.